lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -60,29 +61,30 @@ module CldrRegexpCollection
       IS: /\A\d{3}\z/,
       IN: /\A\d{6}\z/,
       ID: /\A\d{5}\z/,
-      IL: /\A\d{5,7}\z/,
+      IL: /\A\d{5}(\d{2})?\z/,
       JO: /\A\d{5}\z/,
       KZ: /\A\d{6}\z/,
       KE: /\A\d{5}\z/,
       KW: /\A\d{5}\z/,
       LA: /\A\d{5}\z/,
-      LB: /\A(\d{4}([ ]?\d{4})?)?\z/,
-      LU: /\A(L\-)?\d{4}\z/,
+      # NOTE: Lebanon has no postal codes
+      # LB: /\A(\d{4}([ ]?\d{4})?)?\z/,
+      LU: /\A(L|L\-)?\d{4}\z/,
       MK: /\A\d{4}\z/,
       MY: /\A\d{5}\z/,
       MV: /\A\d{5}\z/,
-      MT: /\A[A-Z]{3}[ ]?\d{2,4}\z/,
-      MU: /\A[ARar1-9]\d{4}\z/,
+      MT: /\A[A-Z]{3}[ ]?\d{2,4}\z/i,
+      MU: /\A[AR1-9]\d{4}\z/i,
       MX: /\A\d{5}\z/,
       MA: /\A\d{5}\z/,
       NZ: /\A\d{4}\z/,
       NI: /\A\d{5}\z/,
-      NG: /\A(\d{6})?\z/,
-      OM: /\A(PC )?\d{3}\z/,
+      NG: /\A\d{6}\z/,
+      OM: /\A(PC )?\d{3}\z/i,
```

Comment:
Looks like Oman doesn't use zipcodes: https://worldpostalcode.com/oman/

Again, happy to revert or keep the `i` as well as remove all together if not required.